Azure/azure-cli
# [App Service] Fix #19491: `az functionapp show`: Add kind validation for show operation

**Description**
Fix #19491: `az functionapp show` should not show webapps.  


**Testing Guide**
`az functionapp show -n {webapp}` now returns an error message.
`az webapp show -n {functionapp}` now returns an error message.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
